Tests/test_SeqIO_convert.py

```diff
@@ -199,8 +198,43 @@ def failure_check(self, filename, in_format, out_format, alphabet):
     ("GenBank/NC_005816.gb", "gb", None),
     ("GenBank/cor6_6.gb", "genbank", None),
 ]
+
+formats = (
+    ("genbank", "fasta"),
+    ("gb", "fasta"),
+    ("embl", "fasta"),
+    ("fastq", "fasta"),
+    ("fastq-sanger", "fasta"),
+    ("fastq-solexa", "fasta"),
+    ("fastq-illumina", "fasta"),
+    ("fastq", "tab"),
+    ("fastq-sanger", "tab"),
+    ("fastq-solexa", "tab"),
+    ("fastq-illumina", "tab"),
+    ("fastq", "fastq"),
+    ("fastq-sanger", "fastq"),
+    ("fastq-solexa", "fastq"),
+    ("fastq-illumina", "fastq"),
+    ("fastq", "fastq-sanger"),
+    ("fastq-sanger", "fastq-sanger"),
+    ("fastq-solexa", "fastq-sanger"),
+    ("fastq-illumina", "fastq-sanger"),
+    ("fastq", "fastq-solexa"),
+    ("fastq-sanger", "fastq-solexa"),
+    ("fastq-solexa", "fastq-solexa"),
+    ("fastq-illumina", "fastq-solexa"),
+    ("fastq", "fastq-illumina"),
+    ("fastq-sanger", "fastq-illumina"),
+    ("fastq-solexa", "fastq-illumina"),
+    ("fastq-illumina", "fastq-illumina"),
+    ("fastq", "qual"),
+    ("fastq-sanger", "qual"),
+    ("fastq-solexa", "qual"),
+    ("fastq-illumina", "qual"),
+)
+
```

Comment:
I prefer automatically testing all the combinations over having a hard coded list which must be updated by hand when something new is added.